WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -193,7 +194,12 @@ private async Task<ImmutableArray<AliceClient>> CreateRegisterAndConfirmCoinsAsy
 				}
 			}
 
-			var aliceClients = smartCoins.Select(RegisterInputTask).ToImmutableArray();
+			var aliceRegistrationTasks = smartCoins.Select(RegisterInputTask);
```

Comment:
After this line, all the Tasks are started.

Reply:
what about changing line 202 to improve readability (untested)

```cs
			var aliceClients = Enumerable.Zip(smartCoins, delayTasks, (coin, delay) => delay.Then(RegisterInputTask(coin))).ToImmutableArray();
```

Reply:
I would appreciate it if this would be more trivial, the "how to" is your call.

Reply:
Ok. I will work to make it more obvious.